test/models/audit/ReturnForDataEntryAuditModelSpec.scala

```diff
@@ -169,6 +172,151 @@ class ReturnForDataEntryAuditModelSpec extends SpecBase {
         )
       )
     }
+
```

Comment:
Can we add a test to check multiple corrections, this will check we map over them correctly.

Reply:
Added test to check multiple corrections